static/app/views/insights/pages/backend/laravelOverviewPage.tsx

```diff
@@ -666,9 +674,62 @@ const getCellColor = (value: number, thresholds: Record<string, number>) => {
   return Object.entries(thresholds).find(([_, threshold]) => value >= threshold)?.[0];
 };
 
+const PathCell = styled('div')`
+  display: flex;
+  flex-direction: column;
+  padding: ${space(1)} ${space(2)};
+  justify-content: center;
+  gap: ${space(0.5)};
+
+  [role='columnheader'] & {
+    padding-top: ${space(0.75)};
+    padding-bottom: ${space(0.75)};
+  }
+`;
+
+const ControllerText = styled('div')`
+  color: ${p => p.theme.gray300};
+  font-size: ${p => p.theme.fontSizeSmall};
+  ${p => p.theme.overflowEllipsis};
+  line-height: 1;
+  width: 25vw;
+`;
+
+const Cell = styled('div')`
+  display: flex;
+  align-items: center;
+  gap: ${space(0.5)};
+  overflow: hidden;
+  white-space: nowrap;
+  padding: ${space(1)} ${space(2)};
+
+  &[data-color='danger'] {
+    color: ${p => p.theme.red400};
+  }
+  &[data-color='warning'] {
+    color: ${p => p.theme.yellow400};
+  }
+  &[data-align='right'] {
+    text-align: right;
+    justify-content: flex-end;
+  }
+`;
+
+const HeaderCell = styled(Cell)`
+  padding: ${space(0.25)} ${space(2)};
```

Comment:
The table already wraps the header with an div that has padding. If you add padding here, they are "double padded"
<img width="605" alt="Screenshot 2025-02-27 at 12 09 29" src="https://github.com/user-attachments/assets/aadf54dc-cf5a-4fdf-bb5f-52116c144966" />

```suggestion
  padding: 0;
```